daemon/api.go

```diff
@@ -1852,6 +1852,13 @@ func changeInterfaces(c *Command, r *http.Request, user *auth.UserState) Respons
 	st.Lock()
 	defer st.Unlock()
 
+	for i := range a.Plugs {
+		a.Plugs[i].Snap = systemCoreSnapUnalias(a.Plugs[i].Snap)
```

Comment:
"alias" means something else in the snap world. Perhaps;

- useSnapNick
- dropSnapNick